pyproject.toml

```diff
@@ -1,18 +1,61 @@
+#########################
+# build-system settings #
+#########################
+
 [build-system]
 requires = [
-    "build",
-    "cmake>=3.18",
-    "oldest-supported-numpy",
-    "setuptools",
-    "wheel",
+    "scikit-build-core>=0.8",
+]
+build-backend = "scikit_build_core.build"
+
+
+####################
+# project settings #
+####################
+
+[project]
+name = "python-casacore"
+version = "3.5.3"
+description = "A wrapper around CASACORE, the radio astronomy library"
+keywords = ["pyrap", "casacore", "utilities", "astronomy"]
+license = {file = "LICENSE"}
+readme = {file = "README.rst", content-type = "test/x-rst"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Malte Marquarding", email = "Malte.Marquarding@gmail.com"},
+    {name = "Ger van Diepen", email = "gervandiepen@gmail.com"},
+    {name = "Gijs Molenaar", email = "gijs@pythonic.nl"},
+    {name = "Tammo Jan Dijkema", email = "dijkema@astron.nl"},
+    {name = "Marcel Loose", email = "loose@astron.nl"},
+]
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Natural Language :: English",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: C++",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Astronomy",
+]
+dependencies = [
+    "numpy",
+    "six",
```

Comment:
Are those really the only python dependencies?

Reply:
These were the only ones listed in `setup.py`. This list seems to be complete. A check revealed there's an additional optional dependency on `wxPython` for the table browser.

Reply:
I was not aware that `wxPython` was a dependency. I don't think it should become a required dependency, as the main uses of python-casacore do not need it and the installation overhead is quite big (like requiring x11). So I'd suggest sticking with `numpy` and `six`.